### DISCUSSION TITLE
Welcome to the-calendarium Discussions!

### DISCUSSION THREAD

**Discussion by valentine195** [javalent/calendarium#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rroby1** [comment]
Hello. I'm a new user to Obsidian and Calendarium. This is an incredible plug in! 

An idea I would like to see is the ability to insert the "current" date from the calendar into a note.
